examples/lightclient/lightclient.c

```diff
@@ -514,7 +514,11 @@ int main(int argc, char *argv[])
                 {
                     fprintf(stderr, "Error in recvfrom(): %d %s\r\n", errno, strerror(errno));
                 }
-                else if (0 < numBytes)
+                else if (numBytes >= MAX_PACKET_SIZE) 
+                {
+                    fprintf(stderr, "Received packet >= MAX_PACKET_SIZE\r\n");
+                } 
+                else
```

Comment:
0 length packets are now processed. Are they handled correctly?

Reply:
this was not intended will push an update